apps/ejabberd/src/ejabberd_receiver.erl

```diff
@@ -391,6 +395,8 @@ free_parser(Parser) ->
 gen_server_call_or_noproc(Pid, Message) ->
     try
         gen_server:call(Pid, Message)
-    catch exit:{noproc,Extra} ->
-        {error, {noproc,Extra}}
+    catch exit:{noproc, Extra} ->
+        {error, {noproc, Extra}}
     end.
+
+gen_fsm() -> p1_fsm.
```

Comment:
This warning is quite strange, as this function is used. @gadgetci any comment about this? The function you complain about is used here: https://github.com/esl/MongooseIM/pull/1078/files#diff-94c94811656f8db6c7a9465cde6ea01dR162 for instance.